cookbook/core/control_flow/map_task.py

```diff
@@ -65,5 +65,78 @@ def my_map_workflow(a: typing.List[int]) -> str:
     print(f"{result}")
 
 # %%
-# By default, the map task uses the K8s Array plugin. Map tasks can also run on alternate execution backends, such as `AWS Batch <https://docs.flyte.org/en/latest/deployment/plugin_setup/aws/batch.html#deployment-plugin-setup-aws-array>`__,
+# When defining a map task, avoid calling other tasks in it. Flyte
+# can't accurately register tasks that call other tasks.  While Flyte
+# will correctly execute a task that calls other tasks, it will not be
+# able to give the full performance advantages. This is
+# especially true for map tasks.
+#
+# In this example, the map task ``suboptimal_mappable_task`` would not
+# give you the best performance.
+@task
+def upperhalf(a: int) -> int:
+    return a / 2 + 1
+
+@task
+def suboptimal__mappable_task(a: int) -> str:
+    inc = upperhalf(a=a)
+    stringified = str(inc)
+    return stringified
+
+# %%
+#
+# By default, the map task uses the K8s Array plugin. Map tasks can
+# also run on alternate execution backends, such as
+# `AWS Batch <https://docs.flyte.org/en/latest/deployment/plugin_setup/aws/batch.html#deployment-plugin-setup-aws-array>`__,
 # a provisioned service that can scale to great sizes.
+
+
+# %%
+# Map a task with multiple inputs
```

Comment:
Shouldn't we use the title case?

Reply:
Something like "Map a Task with Multiple Inputs" (titlecaseconverter.com -> Chicago)